packages/story-content-format/src/index.ts

```diff
@@ -116,7 +124,10 @@ export namespace ComposedElement {
             QuoteNode.isQuoteNode(value) ||
             ListNode.isListNode(value) ||
             ListItemNode.isListItemNode(value) ||
-            ListItemTextNode.isListItemTextNode(value)
+            ListItemTextNode.isListItemTextNode(value) ||
+            TableNode.isTableNode(value) ||
+            TableRowNode.isTableRowNode(value) ||
+            TableCellNode.isTableCellNode(value)
```

Comment:
These should also be added to `BlockElement.isBlockNode()`.

Additionally, `Tabe{X}Node.validate{X}Node()` should be added to `BlockElement.validateBlockElement()` and `ComposedElement.validateComposedElement()`.